### PR TITLE
adding images

### PR DESCRIPTION
**Pull request by Maedeh86**
add images

### CODE REVIEW

**Review by Svpriyaa2808 — COMMENTED** [review]
are you adding images alone

**Review by Kavitha0691 — COMMENTED** [review]
we already replace the new images in homepage sidebar and merged to develop branch. are you going to add these images in birds page?

**Comment by Maedeh86** [comment]
i want to added all sounds and images too

File: Program.cs
Comment (Vineethaveerapu) [inline Program.cs:1]: why c# in javascript

File: assignment-3.csproj
Comment (Vineethaveerapu) [inline assignment-3.csproj:1]: i guess this is not relevant

File: birds.html [hunk 1]
```diff
@@ -3,62 +3,40 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Australian zoo</title>
-    <link rel="stylesheet" href="./styles/shared.css" />
-    <link rel="stylesheet" href="./styles/header.css" />
-    <link rel="stylesheet" href="./styles/sidebar.css" />
-    <script src="./js/sidebar.js" defer></script>
-    <script src="./js/header.js" defer></script>
+    <title>Australian Zoo</title>
+    <!-- CSS Files -->
+    <link rel="stylesheet" href="../assignment-3/styles/shared.css" />
```
Comment (Vineethaveerapu) [inline birds.html:8]: i think this path are outside of project,  this will not work

File: birds.html [hunk 2]
```diff
@@ -70,6 +48,92 @@
       </nav>
     </header>
 
-    <!-- header ends -->
+    <!-- Main Container -->
+    <div class="container">
+      <!-- Sidebar -->
+      <aside class="sidebar">
```
Comment (Vineethaveerapu) [inline birds.html:54]: in sub-pages you should only have birds

File: obj/Debug/netcoreapp3.1/assignment-3.csproj.FileListAbsolute.txt
Comment (Vineethaveerapu) [inline obj/Debug/netcoreapp3.1/assignment-3.csproj.FileListAbsolute.txt:1]: can you update gitignore and debug and bin

**Review by Vineethaveerapu — CHANGES_REQUESTED** [review]
i am not understanding the some part of the code, can you please tell

**Review by Vineethaveerapu — APPROVED** [review]
sidebar and main content is working

**Pull request merged**